src/gen.c

```diff
@@ -2139,12 +2139,18 @@ static int btfgen_remap_id(__u32 *type_id, void *ctx)
 /* Generate BTF from relocation information previously recorded */
 static struct btf *btfgen_get_btf(struct btfgen_info *info)
 {
+	const struct btf *base;
```

Comment:
Nit: Let's keep the declarations in this block in reverse-Christmas tree order (longest first) if possible, please.